samples/Sentry.Samples.Android/MainActivity.cs

```diff
@@ -6,7 +6,7 @@ public class MainActivity : Activity
 {
     protected override void OnCreate(Bundle? savedInstanceState)
     {
-        SentrySdk.Init(this, o =>
+        SentrySdk.Init(o =>
```

Comment:
interesting, things work alright like this? I thought this worked on MAUI for example because there's some Init call going on there

Reply:
Originally, yes.  But when I added iOS that changed because there's nothing to pass.

Instead we do this, after we validate the DSN in the main `Init`

https://github.com/getsentry/sentry-dotnet/blob/7ff846f2c15c5d77d3f54e4a7c78a6ddae4abe74/src/Sentry/SentrySdk.cs#L54-L59